test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersAmendHandlerSpec.scala

```diff
@@ -31,21 +30,19 @@ import uk.gov.hmrc.merchandiseinbaggage.model.api.calculation.{CalculationRespon
 import uk.gov.hmrc.merchandiseinbaggage.model.api.payapi.{JourneyId, PayApiResponse}
 import uk.gov.hmrc.merchandiseinbaggage.model.core.{DeclarationJourney, URL}
 import uk.gov.hmrc.merchandiseinbaggage.service.{MibService, PaymentService, TpsPaymentsService}
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub._
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 import uk.gov.hmrc.merchandiseinbaggage.views.html.{CheckYourAnswersAmendExportView, CheckYourAnswersAmendImportView}
 
 import java.time.LocalDateTime
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class CheckYourAnswersAmendHandlerSpec
-    extends DeclarationJourneyControllerSpec
-    with MibConfiguration
-    with PropertyBaseTables {
+class CheckYourAnswersAmendHandlerSpec extends DeclarationJourneyControllerSpec with PropertyBaseTables {
 
   private val importView: CheckYourAnswersAmendImportView = injector.instanceOf[CheckYourAnswersAmendImportView]
   private val exportView: CheckYourAnswersAmendExportView = injector.instanceOf[CheckYourAnswersAmendExportView]
   private val mibConnector: MibConnector                  = injector.instanceOf[MibConnector]
+  private val mibStub: MibBackendStub                     = injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these stub changes, thanks